live/engine/live_exec.py

```
@@ -2349,6 +2393,65 @@ def _fetch_live_balance(funder: str | None = None) -> float | None:
+    try:
+        creds = c.create_or_derive_api_key()
+    except Exception as exc:
+        raise SystemExit(
+            f"Derivation failed: {type(exc).__name__}. The venue throttles this "
+            f"endpoint -- wait a few minutes and run this once more. Everything "
+            f"else keeps working; only account value needs credentials.") from exc
+
+    text = env_file.read_text(encoding="utf-8")
+    lines = [ln for ln in text.splitlines()
+             if not ln.split("=", 1)[0].strip() in
+             ("POLY_API_KEY", "POLY_API_SECRET", "POLY_API_PASSPHRASE")]
```

Ruff (0.16.1)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)